### ISSUE TITLE
Restore the "network security" saved queries

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2918]
## tl;dr

The built-in set of network security queries that came by default with the older Brim app were removed from Zui in #2415. Recent experiences with the community have led us to consider restoring them somehow.

## Details

At the time this issue is being opened, Zui is at commit c437661.

As #2415 was being merged, we were in the middle of making Zui more approachable for general data shaping/querying and not just the DFIR workflows the older Brim app was best known for. As part of this, we were concerned that these new users might be confused if the built-in Zui queries were all specialized for a "network security" use case that might be foreign to them. Therefore a decision was made to no longer bundle them by default and instead have them available in the Brimcap repo along with [instructions](https://github.com/brimdata/brimcap#brimcap-queries) on how to download and install them.

To help users navigate that change, the [Zui docs](https://zui.brimdata.io/docs) make a point of mentioning Brimcap in a handful of locations in hopes that users will notice this and find the query set as they're learning about how pcap processing is handled in Zui. However, we've recently had a couple community inquiries from users that know about the queries (either because they were former users of the Brim app or they find reference to them in some of our older blog/video materials.

For example, there was [this recent Tweet](https://twitter.com/FerryTuckwit/status/1725945448039317964):

![image](https://github.com/brimdata/zui/assets/5934157/0814105b-87b1-4da6-94e3-ec7d14080479)

Or [this recent community Slack message](https://brimdata.slack.com/archives/CTT24051C/p1701304911677979):

![image](https://github.com/brimdata/zui/assets/5934157/7ddb0b4f-e4b0-4744-ab19-3757c2b22a06)

There's also [this other community Slack thread](https://brimdata.slack.com/archives/C010DR0HHMF/p1700305388493329?thread_ts=1699776926.923289&cid=C010DR0HHMF):

![image](https://github.com/brimdata/zui/assets/5934157/104a00f7-f2ad-4ce2-b560-2c2c92ad248e)

When this happens, it's not difficult for us to send them the link to https://github.com/brimdata/brimcap#brimcap-queries and get them on their way. However, the fact that this has happened multiple times has us thinking we could take more steps to avoid them getting slowed down having to ask. We discussed as a group, and here's a few ideas that were kicked around.

1. **We could simply start including them by default again**. When we merged #2415, it wasn't clear how long the security users would continue loving Zui for those traditional use cases, but it seems the people still dig it, which is a nice problem to have. 😄  If we do restore it, I'd personally suggest a different name for the folder such as "Network Security" since the folder name "Brimcap" that's been used for the manually-installed query set reads more like a "how" (these are queries associated with a pcap processing tool called Brimcap) rather than the "what" (what are these queries used for? network security!)

2. **Have a way they can be loaded on-demand.** While there was consensus on including them again for brand new app installs, @jameskerr made the point that he'd probably not make the app add them automatically as part of an upgrade (since this can be messy, e.g., what happens if they have a folder that already exists that clashes with the name?) Therefore we might want to have some action that can be taken in the app to load them. A pull-down menu option was suggested, but I have some concerns with that idea since I'm not sure if users always think to check those menus. Personally I think some kind of button or other control in the left-sidebar **QUERIES** area would definitely cover it since a user that goes there looking for them would see that button instead of the empty space that's there now. Regardless, I'm sure @jameskerr will come up with a solid design. 😄

3. **Make one or more other "query kits" available at the same time.** To make it less awkward to have "network security" as the one and only built-in query set, it would be great if we added others at the same time. I'm reminded of existing issue #2388 that proposed adding some built-in sample data and a set of queries to go with it.
